### PR TITLE
Make improvements to building a data service guide

### PR DESCRIPTION
**Pull request by kaneeldias**
## Purpose
$subject

Fixes: https://github.com/ballerina-platform/ballerina-dev-website/issues/4880

### CODE REVIEW

File: learn/get-started/build-a-data-service-in-ballerina.md [hunk 1]
```diff
@@ -124,16 +124,24 @@ The MySQL driver JAR is necessary to connect to and interact with a MySQL server
...
+   
+   # In Windows operating systems
+   [[platform.java11.dependency]]
+   path = "path\to\mysql\driver.jar”
    ```
    
 ### Define MySQL configurations
 
-In the package directory, create a new file named `Config.toml` and specify the configurations below to connect to the MySQL database.
+In the newly created directory `"data_service"` (using the `bal new` command), create a new file named `Config.toml` and specify the configurations below to connect to the MySQL database.
```
Comment (praneesha) [inline learn/get-started/build-a-data-service-in-ballerina.md:144]:
```suggestion
In the  `data_service` directory (you created above using the `bal new` command), create a new file named `Config.toml`, and specify the configurations below to connect to the MySQL database.
```

File: learn/get-started/build-a-data-service-in-ballerina.md [hunk 2]
```diff
@@ -205,6 +212,9 @@ The `mysql:Client` provides two primary remote methods for performing queries.
 To use the `query()`, `queryRow()`, and `execute()` methods, which can perform basic CRUD operations against the MySQL database, add the code below to the `main.bal` file. 
 
 ```ballerina
+import ballerina/sql;
```
Comment (praneesha) [inline learn/get-started/build-a-data-service-in-ballerina.md:215]:
```suggestion
import ballerina/sql;
```
Don't we need to remove this unused import?

File: learn/get-started/build-a-data-service-in-ballerina.md [hunk 3]
```diff
@@ -471,15 +481,16 @@ bal run
 You view the output below.
 
 >**Info:** This creates an `/employees` endpoint on port `8080`, which can 
-be accessed via a browser by visiting `http://locahost:8080/employees`.
+be accessed via a browser by visiting `http://localhost:8080/employees`.
```
Comment (praneesha) [inline learn/get-started/build-a-data-service-in-ballerina.md:484]:
```suggestion
be accessed by visiting `http://localhost:8080/employees`.
```

File: learn/get-started/build-a-data-service-in-ballerina.md [hunk 4]
```diff
@@ -471,15 +481,16 @@ bal run
...
 
 ## Try the service
 
 Invoke the defined resource function by sending the `POST` request below to `http://localhost:8080/employees` with the required data as a JSON payload.
 
+>**Info:** The following CURL command only works with Unix operating systems.
```
Comment (praneesha) [inline learn/get-started/build-a-data-service-in-ballerina.md:490]:
```suggestion
>**Info:** The following cURL command only works with Unix operating systems.
```
Can we give the command for Windows as well in the info section saying, "If you are using the Windows OS, execute this cURL command: <command>."

File: learn/get-started/build-a-data-service-in-ballerina.md (hunk 2 above)
Comment (kaneeldias) [reply learn/get-started/build-a-data-service-in-ballerina.md:215]: The unused import was in a previous sample. It has been removed from there.

https://github.com/ballerina-platform/ballerina-dev-website/pull/5037/files#diff-439f92e0c6479c601f714f2743e690239e2745cf2eaecd4de77817f7c9ad9836L169

File: learn/get-started/build-a-data-service-in-ballerina.md (hunk 3 above)
Comment (kaneeldias) [reply learn/get-started/build-a-data-service-in-ballerina.md]: Fixed in [acdbdda](https://github.com/ballerina-platform/ballerina-dev-website/pull/5037/commits/acdbdda187290416c649754ed6c66fceeefc2dca)

File: learn/get-started/build-a-data-service-in-ballerina.md (hunk 2 above)
Comment (kaneeldias) [reply learn/get-started/build-a-data-service-in-ballerina.md:215]: Fixed in [acdbdda](https://github.com/ballerina-platform/ballerina-dev-website/pull/5037/commits/acdbdda187290416c649754ed6c66fceeefc2dca)

File: learn/get-started/build-a-data-service-in-ballerina.md (hunk 1 above)
Comment (kaneeldias) [reply learn/get-started/build-a-data-service-in-ballerina.md]: Fixed [acdbdda](https://github.com/ballerina-platform/ballerina-dev-website/pull/5037/commits/acdbdda187290416c649754ed6c66fceeefc2dca)

**Pull request closed without merging**